.gitignore

```
@@ -164,3 +164,6 @@ target/
 
 # Avoid ignoring Gradle wrapper jar file (.jar files are usually ignored)
 !gradle-wrapper.jar
+
+# VSCode
+.vscode/settings.json
```

```suggestion
.vscode
```

I believe we can ignore the whole `.vscode` path for now.